### PR TITLE
remove old usage of ERR_OUT_OF_MEMORY with memory handling ported to C++

### PR DESCRIPTION
**Pull request by devnexen**

### CODE REVIEW

File: src/errors.h [hunk 1]
```diff
@@ -9,8 +9,7 @@
 #define ERR_VIDEO_COLOUR_DEPTH   6
 #define ERR_AUDIO_INIT           7
 #define ERR_AUDIO_RATE           8
-// TODO: Progressively get rid of ERR_OUT_OF_MEMORY as we replace malloc with new
-#define ERR_OUT_OF_MEMORY        9
+//#define ERR_OUT_OF_MEMORY      9   no longer used
```
Comment (ColinPitrat) [inline src/errors.h:12]: Just remove it completely

Comment (devnexen) [reply src/errors.h:12]: sure, it was not to "put off" with a missing value :)

Comment (ColinPitrat) [reply src/errors.h]: History of the file will document why the gap is here :)

**Comment by ColinPitrat** [comment]
Thanks!

**Pull request merged**